test/examples/to-fix/michelson/Datatypes.ju

```diff
@@ -44,4 +75,32 @@ let main =
   --  Non-exhaustive patterns in Ann _ (Pi argUsage _ _) (LamM _ [name] body)
   let pm = patternMatch (Q 0 1 2) in
 
+  -- Typechecks, but fails to compile:
+  --  juvix: src/Juvix/Backends/Michelson/Compilation.hs:53:13-75:
+  --  Non-exhaustive patterns in Ann _ (Pi argUsage _ _) (LamM _ [name] body)
+  let ftict = fticTestTerm in
+
+  -- Typechecks, but fails to compile:
+  --  juvix: src/Juvix/Backends/Michelson/Compilation.hs:53:13-75:
+  --  Non-exhaustive patterns in Ann _ (Pi argUsage _ _) (LamM _ [name] body)
+  let sexpTest = ssize s1 in
```

Comment:
Yeah, currently we only compile Paired functions for Michelson, as those are the valid contract types